.github/workflows/pull-request.yml

```
@@ -432,11 +433,72 @@ jobs:
+      - name: Setup Git Config
+        run: |
+          if ! git config user.name "GitHub Actions Bot" || \
+             ! git config user.email "41898282+github-actions[bot]@users.noreply.github.com"; then
+            echo "Failed to configure Git"
+            exit 1
+          fi
+          
+      - name: Commit changes
+        env:
+          HEAD_REF: ${{ github.head_ref }}
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)